server/container_create.go

```diff
@@ -27,6 +27,9 @@ import (
 	"github.com/pkg/errors"
 )
 
+// sync with https://github.com/containers/storage/blob/7fe03f6c765f2adbc75a5691a1fb4f19e56e7071/pkg/truncindex/truncindex.go#L92
+const noSuchID = "no such id"
```

Comment:
Why can't we have the storage code return this typed error instead - https://github.com/containers/storage/blob/7fe03f6c765f2adbc75a5691a1fb4f19e56e7071/pkg/truncindex/truncindex.go#L23?

Reply:
easier to backport this :) I planned on updating the error in main